addons/web/models/ir_http.py

```diff
@@ -41,6 +41,7 @@ def session_info(self):
             "company_id": user.company_id.id if request.session.uid else None,  # YTI TODO: Remove this from the user context
             "partner_id": user.partner_id.id if request.session.uid and user.partner_id else None,
             "web.base.url": self.env['ir.config_parameter'].sudo().get_param('web.base.url', default=''),
+            "max_file_size": int(self.env['ir.config_parameter'].sudo().get_param('web.max_file_size', default=128 * 1024 * 1024)),  # 128MiB
```

Comment:
Could we make the key slightly more specific? (also computing the default size on a different line could make the line shorter and the code more readable)

```suggestion
            "max_file_size": int(self.env['ir.config_parameter'].sudo().get_param('web.max_file_upload_size', default=128 * 1024 * 1024)),  # 128MiB
```